### PR TITLE
Update cli.py

### PR DESCRIPTION
**Pull request by henrymbuguakiarie**
## Purpose
I've added a named snippet section within the code for out-of-repo snippet references.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by henrymbuguakiarie** [comment]
Hello @ckittel @mmacy Please review

**Review by ckittel — APPROVED** [review]
LGTM

File: src/cli-access-protected-api/cli.py [hunk 1]
```diff
@@ -70,6 +71,7 @@ def main(tenant: str, client_id: str, object_id: str, secret: str):
     )
 
     # First check for an existing token in the cache and/or refresh if needed
+    # <ms_docref_get_graph_token>
```
Comment (mmacy) [inline src/cli-access-protected-api/cli.py:74]: @henrymbuguakiarie Please include descriptive comments like this in the named snippets.

Cc: @ckittel @Dickson-Mwendia

**Review by mmacy — COMMENTED** [review]
@henrymbuguakiarie Missed the merge, but please take note of this comment.

FYI for @ckittel for future reviews and @Dickson-Mwendia for future out-of-repo-code snippet (OORCS) integration.

**Pull request merged**